app/services/hyrax/repository_fixity_check_service.rb

```diff
@@ -1,6 +1,5 @@
 module Hyrax
   class RepositoryFixityCheckService
-    # TODO: this never seems to be used. Remove?
     def self.fixity_check_everything
```

Comment:
There is this: https://github.com/samvera/hyrax/wiki/Hyrax-Management-Guide#fixity-checking Good enough?